main/integration-tests/support/resources/studies/study-upload-requests.js

```diff
@@ -30,6 +30,9 @@ class StudyUploadRequests extends Resource {
   }
 
   // ************************ Helpers methods ************************
+  async getPresignedRequests(filenames = []) {
```

Comment:
Would you mind adding this helper method to the spreadsheet and to the slack channel thread?